desc/continuation.py

```diff
@@ -373,7 +373,7 @@ def _add_shaping(
                 None,
                 eqi,
                 bdry_ratio,
-                1,
+                _get_ratio(eqi.pressure, eq.pressure),
```

Comment:
I remember this was a somewhat deliberate choice to hard code this to 1, since otherwise sometimes it would print 0.99999... for the last iteration when we know it should be 1